UefiCpuPkg/PiSmmCpuDxeSmm/CpuS3.c

```diff
@@ -233,7 +231,6 @@ InitSmmS3ResumeState (
     }
 
     SmmS3ResumeState->SmmS3Cr0 = (UINT32)AsmReadCr0 ();
-    SmmS3ResumeState->SmmS3Cr3 = Cr3;

```

Comment:
Jiaxin, can you remove mSmmS3ResumeState?

Reply:
synced with ray, keep the mSmmS3ResumeState since it's needed.